app/views/leaderboard.scala.html

```diff
@@ -145,3 +165,14 @@ <h5>@Messages("leaderboard.weekly.detail")</h5>
         </div>
     }
 </div>
+
+<h4 style="text-align: center">
```

Comment:
Can you add this style stuff to main.css next to the rest of the leaderboard stuff? You'll probably need to add an `id` for the header.